app/build.gradle

```diff
@@ -30,6 +30,7 @@ android {
     packagingOptions {
         exclude 'META-INF/rxjava.properties'
     }
+    buildToolsVersion '26.0.2'
```

Comment:
This line is duplicated above -- AS probably added automatically because it didn't recognize the `versions.buildTools` ext but it should be okay to remove